File: 2160-minimum-sum-of-four-digit-number-after-splitting-digits/2160-minimum-sum-of-four-digit-number-after-splitting-digits.py
class Solution:
    def minimumSum(self, num: int) -> int:
        s = []
        while num > 0:
            s.append(num % 10)
            num //= 10
        s.sort()
        return s[0] * 10 + s[1] * 10 + s[2] + s[3]

File: 2160-minimum-sum-of-four-digit-number-after-splitting-digits/2160-minimum-sum-of-four-digit-number-after-splitting-digits.java
class Solution {
    public int minimumSum(int num) {
        List<Integer> s = new ArrayList<>();
        while (num > 0) {
            s.add(num % 10);
            num /= 10;
        }
        Collections.sort(s);
        return s.get(0) * 10 + s.get(1) * 10 + s.get(2) + s.get(3);
    }
}